pkg/cluster/clusterLeader.go

```diff
@@ -65,9 +65,7 @@ func NewManager(path string, inCluster bool) (*Manager, error) {
 			return nil, err
 		}
 
-		// TODO - we need to make the host/port configurable
-
-		config.Host = fmt.Sprintf("%s:6443", id)
+		config.Host = fmt.Sprintf("%s:%v", id, port)
```

Comment:
Should this be `%d` for the port?

Reply:
%v is the value in the default format so it works fine